doc/classes/DisplayServer.xml

```diff
@@ -666,6 +667,26 @@
 			<argument index="0" name="mode" type="int" enum="DisplayServer.WindowMode" />
 			<argument index="1" name="window_id" type="int" default="0" />
 			<description>
+				Sets window mode for the given window to [code]mode[/code].
+				[codeblocks]
+				[gdscript]
+				# Set window mode to:
+				DisplayServer.window_set_mode(0) # Windowed
+				DisplayServer.window_set_mode(1) # Minimized
+				DisplayServer.window_set_mode(2) # Maximized
+				DisplayServer.window_set_mode(3) # Fullscreen
```

Comment:
We don't need a code sample here, as usage becomes obvious as soon as you look at the WindowMode constant.

Reply:
I didn't realize at first that you could link the enum, done.